### PR TITLE
'Create new Report' Page

### PR DESCRIPTION
**Pull request by heupotato**
- [x] Created "Create new Report" Page 
image
- [x] Updated navigations
- [x] Fixed  previous PR's comment and other small issues

### CODE REVIEW

**Comment by heupotato** [comment]
ready

File: DailyReport/src/main/java/app/controller/ReportController.java [hunk 1]
```diff
@@ -56,4 +61,47 @@ public ModelAndView showReport(@PathVariable("id") int id, HttpSession session)
...
+		report.setCreatedAt(DateUtils.getToday());
+		
+		report.setActure(messageSource.getMessage("report.acture.default", null, Locale.US));
+		
+		model.addObject("currentUser", session.getAttribute("currentUser")); 
+		model.addObject("report", report); 
+		return model; 
+	}
+	
+	@RequestMapping(value = "/reports/save", method = RequestMethod.POST)
```
Comment (vanvtt-0952) [inline DailyReport/src/main/java/app/controller/ReportController.java:82]:
```suggestion
	@RequestMapping(value = "/reports/", method = RequestMethod.POST)
```

Comment (heupotato) [reply DailyReport/src/main/java/app/controller/ReportController.java:82]: Em đã fix tại branch update-report rồi ạ

**Comment by heupotato** [comment]
merge

**Pull request merged**